Gopkg.lock

```diff
@@ -0,0 +1,51 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
+
+
+[[projects]]
+  digest = "1:bb1449042232e17167a8ed054128ae9f41f2ccdb7e3956903494524b946bab23"
+  name = "github.com/bitrise-io/go-utils"
+  packages = [
+    "colorstring",
+    "command",
+    "errorutil",
+    "fileutil",
+    "log",
+    "pathutil",
+    "retry",
+  ]
+  pruneopts = "UT"
+  revision = "6eb582e08179a025ed2deb86ea6dffbd13ba1ff6"
```

Comment:
Is this the master branch's head?

Reply:
Yes, it is:
https://github.com/bitrise-io/go-utils/commit/6eb582e08179a025ed2deb86ea6dffbd13ba1ff6

Go packages don't have releases this is why it's locked on a commit hash.